RecoMuon/MuonSeedGenerator/src/RPCSeedGenerator.cc

```diff
@@ -241,6 +241,7 @@ RPCSeedGenerator::produce(edm::Event& iEvent, const edm::EventSetup& iSetup)
 
     // Start from filling layers to filling seeds
     LayerFinder.fill();
+    Overlapper.setEventSetup(iSetup);
```

Comment:
there is a call to setEventSetup in the ```beginRun```.
Is it added here because iSetup can now change during a job (or an IOV)?

Reply:
The `edm::EventSetup` object is now a temporary that only lives for the life-time of the call. All the management of the IOVs and data is done by a new `edm::EventSetupImpl` object which is held by the `edm::EventSetup` and has a longer lifetime.